src/api/users.js

```diff
@@ -0,0 +1,13 @@
+import { client } from "./httpClient";
+
+export const getUsers = () => {
+  return client.get("/users");
+};
+
+export const getUserByEmail = (email) => {
+  return client.get(`/users?email=${email}`);
+};
+
+export const registrationUser = (email, name) => {
+  return client.post(`/users?email=${email}`, { email, name });
```

Comment:
It seems like there is a misunderstanding in how to use HTTP methods. The POST method is used to send data to the server to create a new resource. The query parameters (like `email=${email}`) are usually used with GET requests to filter the returned data. In your case, you are trying to create a new user, so the data should be sent in the body of the request, not in the query parameters. The correct usage would be `client.post('/users', { email, name })`.